integration/spark2/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -533,6 +533,7 @@ object AlterTableUtil {
       "COMMENT",
       "COLUMN_META_CACHE",
       "CACHE_LEVEL",
+      "COMPACTION_LEVEL_THRESHOLD",
```

Comment:
Here it is added as a table properties right ? Need to add validation also ?
I guess now it will accept any value.

Reply:
solved